### DISCUSSION TITLE
sf 1.10.1

### DISCUSSION THREAD

**Discussion by richardlehane** [richardlehane/siegfried#232] [General]
@ross-spencer @prettybits I'm going to try to pull together the fixes (thanks for those!) for some of the bugs that surfaced with the 1.10 release to cut a new point release over the weekend.

Please let me know if you need me to wait for anything or if there's anything you'd like included

**Comment by ross-spencer** [comment]
Thanks @richardlehane. Don't think I've any contributions for this one.

**Comment by prettybits** [comment]
@richardlehane Nothing more than what's already there from me for this point release, unless you need further changes that you don't prefer to do yourself. Thanks!
